### PR TITLE
bug removed

### PR DESCRIPTION
**Pull request by Bhany**
.

### CODE REVIEW

File: 2021/day1.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
         pre = curr
         curr = file.readline()
 
-# -----------------------------------
+# ----------------------------------
```
Comment (hhj0319) [inline 2021/day1.py:12]: Nice fix

File: 2021/day1.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
 # Part 2
```
Comment (hhj0319) [inline 2021/day1.py]: You can fix this too!

**Review by hhj0319 — CHANGES_REQUESTED** [review]

File: 2021/day1.py (hunk 2 above)
Comment (Bhany) [reply 2021/day1.py]: okat

File: 2021/day1.py [hunk 3]
```diff
@@ -9,8 +9,8 @@
         pre = curr
         curr = file.readline()
 
-# -----------------------------------
-# Part 2
+# ----------------------------------
+# Part. 2
```
Comment (hhj0319) [inline 2021/day1.py:13]: Nice

**Review by hhj0319 — APPROVED** [review]

**Pull request merged**